Packs/SlackITAdmin/Integrations/Slack_IT_Admin/Slack_IT_Admin.py

```diff
@@ -0,0 +1,724 @@
+                                            active=res_json.get('active'), details=res_json)
+    elif res.status_code == 404:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=404,
+                                            errorMessage=res_json.get('errorSummary'), details=res_json)
+    else:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=res_json.get('errorCode'),
+                                            errorMessage=res_json.get('errorSummary'), details=res_json)
+
+    generic_iam_context_dt = f'{generic_iam_context.command}(val.id == obj.id && val.instanceName == obj.instanceName)'
+
+    outputs = {
+        generic_iam_context_dt: generic_iam_context.data
+    }
+
+    readable_output = tableToMarkdown(name=f'{format_pre_text} Slack User:',
+                                      t=generic_iam_context.data,
+                                      headers=["brand", "instanceName", "success", "active", "id", "username", "email",
+                                               "errorCode", "errorMessage", "details"],
+                                      removeNull=True)
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def get_group(client, args):
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    res = client.get_group_by_id(group_id)
+    res_json = res.json()
+
+    if res.status_code != 200:
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {res_json}")
+
+    members = res_json.get('members')
+    outputs = {
+        'Slack.Group(val.Id && val.Id === obj.Id)': {
+            'Id': group_id,
+            'DisplayName': res_json.get('displayName'),
+            'Members': members
+        }
+    }
+
+    readable_output = tableToMarkdown(name=f"Slack Group Members: {res_json.get('displayName')} [{group_id}]",
+                                      t=members)
+
+    return (
+        readable_output,
+        outputs,
+        res_json
+    )
+
+
+def delete_group_command(client, args):
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    res = client.delete_group(group_id)
+
+    if res.status_code != 204:
+        error_json = res.json()
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {error_json}")
+
+    readable_output = f'Slack Group: "{group_id}" was deleted successfully'
+    return (
+        readable_output,
+        {},
```

Comment:
please add to the context the group data (which in this case will be the group name/id) `"deleted": True` (and for other commands `"deleted": False`) and update in the context output in the yml as well.
The output object should look similar to what I described in my comment in get_group() method.